Whiskey/Whiskey.psd1

```diff
@@ -142,7 +142,7 @@
 
             # ReleaseNotes of this module
             ReleaseNotes = @'
-* Added: PublishProGetAsset task that adds and updates assets in ProGet.
+* fixed: Removed ApiKey as requirement for Publish-WhiskeyProGetAsset.

```

Comment:
Capitalize the `f`: `Fixed` instead of `fixed`.

Reply:
Done